airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py

```diff
@@ -881,6 +881,8 @@ def _extract_primary_key_value(record: Mapping[str, Any], primary_key: List[List
     return pk_values
 
 
+@pytest.mark.setup
+@pytest.mark.teardown
```

Comment:
Does these marker mean that the setup and teardown logic will run for each test in the class or once per class instantiation?

Reply:
This marks each test in the class as having an option to do setup/teardown. When the `setup_and_teardown` fixture is being called for each test, it uses the presence of the marker to decide whether to run setup/teardown.